modules/EvseSecurity/main/evse_securityImpl.cpp

```diff
@@ -121,6 +121,26 @@ evse_securityImpl::handle_get_leaf_certificate_info(types::evse_security::LeafCe
     return response;
 }
 
+types::evse_security::GetCertificateFullInfoResult
+evse_securityImpl::handle_get_all_valid_certificates_info(types::evse_security::LeafCertificateType& certificate_type,
```

Comment:
arguments could be const I think

Reply:
By the way, that is auto-generated code. It is the case of all the headers, where they are not passed as const. Any method I can tell the code generator to make those 'const'?

Reply:
As this doesn't seem clear with the code-gen at the moment, will leave as-is.